tests/nemo_text_processing/de/test_sparrowhawk_normalization.sh

```diff
@@ -1,7 +1,7 @@
 #! /bin/sh
 
 GRAMMARS_DIR=${1:-"/workspace/sparrowhawk/documentation/grammars"}
-PROJECT_DIR=${2:-"/workspace/tests/en"}
+PROJECT_DIR=${2:-"/workspace/tests/"}
```

Comment:
Ditto

Reply:
Right, same thing.  Once I removed it, I was able to run Sparrowhawk tests just fine.